### PR TITLE
Storage module updates

### PR DESCRIPTION
**Pull request by dimitrov-d**
- Added file.delete() method
- Added bucket.directory() method
- Added directory.delete() method
- Updated bucket endpoints and JSDoc comments
- IPNS SDK method and CLI commands

### CODE REVIEW

File: packages/sdk/src/modules/storage/directory.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class Directory extends ApillonModel {
   ) {
     super(directoryUuid);
     this.bucketUuid = bucketUuid;
-    this.API_PREFIX = `/storage/${bucketUuid}`;
+    this.API_PREFIX = `/storage/buckets/${bucketUuid}/directories/${directoryUuid}`;
```
Comment (MoMannn) [inline packages/sdk/src/modules/storage/directory.ts:57]: Why is this not: `this.API_PREFIX = /storage/buckets/${bucketUuid}` Then the prefix used in get? I know that base here is directory but if you need a lower level prefix for an api call it should probably stay as that lower level.

File: packages/sdk/src/modules/storage/directory.ts [hunk 2]
```diff
@@ -67,7 +67,7 @@ export class Directory extends ApillonModel {
     this.content = [];
     params.directoryUuid = this.uuid;
     const url = constructUrlWithQueryParams(
-      `${this.API_PREFIX}/content`,
```
Comment (MoMannn) [inline packages/sdk/src/modules/storage/directory.ts:70]: So why is prefix not used here?

**Review by MoMannn — APPROVED** [review]

**Review by MoMannn — APPROVED** [review]

File: packages/cli/README.md [hunk 3]
```diff
@@ -594,23 +773,24 @@ jobs:
...
       # - name: Build app
       #   run: npm run build
 
-
       - name: Deploy website
         env:
           APILLON_API_KEY: ${{ secrets.APILLON_API_KEY }}
           APILLON_API_SECRET: ${{ secrets.APILLON_API_SECRET }}
           WEBSITE_UUID: ${{ secrets.WEBSITE_UUID }}
-        run: npx --yes @apillon/cli hosting deploy-website ./dist --uuid $WEBSITE_UUID --key $APILLON_API_KEY --secret $APILLON_API_SECRET
```
Comment (tinemlakar) [inline packages/cli/README.md:613]: this should be fine as it was!

File: packages/cli/src/modules/storage/ipns.commands.ts [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+    .action(async function () {
+      await listIpnsNames(this.optsWithGlobals());
+    });
+
+  ipns
+    .command('create')
+    .description('Create a new IPNS record for this bucket')
+    .requiredOption('-n, --name <name>', 'Name of the IPNS record')
+    .option('-d, --description <description>', 'Description of the IPNS record')
+    .option('-c, --cid <cid>', 'CID to which this IPNS name will point')
```
Comment (tinemlakar) [inline packages/cli/src/modules/storage/ipns.commands.ts:30]: description could be just "Targeted CID"

File: packages/cli/src/modules/storage/ipns.commands.ts [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+      await getIpns(this.optsWithGlobals());
+    });
+
+  ipns
+    .command('publish')
+    .description('Publish an IPNS record to IPFS and link it to a CID')
+    .requiredOption('-i, --ipns-uuid <uuid>', 'UUID of the IPNS record')
+    .requiredOption(
+      '-c, --cid <string>',
+      'CID to which this IPNS name will point',
```
Comment (tinemlakar) [inline packages/cli/src/modules/storage/ipns.commands.ts:49]: Same as above

File: packages/sdk/README.md [hunk 6]
```diff
@@ -67,101 +70,157 @@ import {
   DeploymentStatus,
   Hosting,
   LogLevel,
-} from '@apillon/sdk';
-import * as fs from 'fs';
+} from "@apillon/sdk";
```
Comment (tinemlakar) [inline packages/sdk/README.md:73]: Why double quotes?!

File: packages/sdk/README.md [hunk 7]
```diff
@@ -67,101 +70,157 @@ import {
...
+await ipns.delete();
+```
 
 ## NFTs
 
 NFT module encapsulates functionalities for NFT service available on Apillon dashboard.
 
+For detailed NFT SDK method, class and property documentation visit [SDK NFT docs](https://sdk-docs.apillon.io/classes/Nft.html).
+
+::: warning
```
Comment (tinemlakar) [inline packages/sdk/README.md:220]: thos should be properly formatted for markdown. For example with md quote:

```
> ***Warning***
> this is the warning!
```

> ***Warning***
> this is the warning!

**Review by tinemlakar — CHANGES_REQUESTED** [review]

**Review by tinemlakar — APPROVED** [review]

**Pull request merged**